src/jit/compiler.cpp

```diff
@@ -4963,6 +4982,10 @@ void Compiler::compCompileFinish()
         // Make the updates.
         genMemStats.nraTotalSizeAlloc = compGetAllocator()->getTotalBytesAllocated();
         genMemStats.nraTotalSizeUsed  = compGetAllocator()->getTotalBytesUsed();
+#if MEASURE_MEM_ALLOC
```

Comment:
Yes, I moved it and spotted the duplication just this morning (and was waiting for feedback before pushing a fix). Thanks for catching that.